development/guideline/communication.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Communication guidelines
+tags: [guidelines, development]
+---
+
+# Communication guidelines
+
+To communicate with and reach a wide audience, we use the following channels
+
+- this website, for documentation
+- the email [discussion list](/discussion_list), for interactions between users and support
+- [GitHub](http://github.com/fieldtrip), for interaction between developpers
+- [Twitter](http://twitter.com/fieldtriptoolbx), for announcements and news
+- [FaceBook](http://facebook.com/fieldtriptoolbox), which we are not actively using at the moment
```

Comment:
Perhaps move this to a separate item, because above it suggests that this is a 'used' channel, which it actually is not.

Reply:
We can also just leave FB out here since there is indeed no strategy for it.